docs/src/lib/functions.md

```diff
@@ -16,6 +16,7 @@ by
 combine
 groupby
 groupindices
+groupvalues
```

Comment:
Thank you for adding this. Also it would be great to add an entry to docs/src/lib/indexing.md with `GroupedDataFrame` section for `getindex` as now it becomes complex enough so that it should be precisely documented.

Reply:
Can you drop this function for now? If we define `keys`, it's probably not necessary anymore, right?

Reply:
this is what I also suggested.

Reply:
Dropped it.